internal/names.go

```diff
@@ -69,6 +69,11 @@ const (
 	// DockerInspectFormatEnvVar is the environment variable set by the CNAB runtime to select
 	// the inspect output format.
 	DockerInspectFormatEnvVar = "DOCKER_INSPECT_FORMAT"
+
+	// LabelAppNamespace is the label used to track app resources
+	LabelAppNamespace = "com.docker.app.namespace"
```

Comment:
```suggestion
	LabelAppNamespace = Namespace + "namespace"
```

Same for the `LabelAppVersion`, you can concat it with the namespace.